app/model/Common/File.php

```diff
@@ -31,6 +33,13 @@ public function getFileName() : string
         return basename($this->path->getPath());
     }
 
+    public function getFileNameWithoutPrefix() : string
```

Comment:
Tohle je potřeba jen pro testy, tak to je zbytečný to sem přidávat. A ve výsledku to ani není potřeba - ten test se dá zjednodušit, viz komentář na https://github.com/skaut/Skautske-hospodareni/pull/1175/files/e8ad5db5e620c1977ec44bb4b09adf98c1cae660..658f01e4575b91126bdc8647436f4a253ca52034#diff-185cd03d0e4313baf82bdeb0bb0f7079R81

Reply:
Já to tu měl shodou okolností nachystané, protože to chci poslat v dalším PR. Chci zobrazovat ve vypisu souborů původní jméno.

Reply:
V tom případě je to ok.